core/runtime/src/main/scala/com/fortysevendeg/exercises/MethodEval.scala

```diff
@@ -65,8 +65,8 @@ class MethodEval() {
   val timeout = 20.seconds
   private val evaluator = new Evaluator(timeout)
 
-  def eval[T](qualifiedMethod: String, rawArgs: List[String], imports: List[String] = Nil): EvaluationResult[_] = {
-    val pre = imports.mkString(System.lineSeparator)
+  def eval[T](pkg: String, qualifiedMethod: String, rawArgs: List[String], imports: List[String] = Nil): EvaluationResult[_] = {
+    val pre = (s"import $pkg._" :: imports).mkString(System.lineSeparator)
```

Comment:
I think `import a._` makes more sense because we could hide Helper to the user using `private[stdlib]` for example